internal/providers/terraform/azure/event_hubs_namespace.go

```diff
@@ -0,0 +1,164 @@
+package azure
+
+import (
+	"fmt"
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/infracost/infracost/internal/usage"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+	"strings"
+)
+
+func GetAzureRMEventHubsNamespaceRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_eventhub_namespace",
+		RFunc: NewAzureRMEventHubs,
+	}
+}
+
+func NewAzureRMEventHubs(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var events, retention, capacityUnit *decimal.Decimal
+	var capacity int64
+	sku := "Basic"
+	meterName := "Throughput Unit"
+	region := lookupRegion(d, []string{})
+
+	if d.Get("sku").Type != gjson.Null {
+		sku = d.Get("sku").String()
+	}
+	if d.Get("capacity").Type != gjson.Null {
+		capacity = d.Get("capacity").Int()
+		capacityUnit = decimalPtr(decimal.NewFromInt(capacity))
+	}
+	if d.Get("auto_inflate_enabled").Type != gjson.Null {
```

Comment:
Why is this TF param checked? Does it signify that this is a dedicated or premium tier? I couldn't see anything in https://azure.microsoft.com/en-in/pricing/details/event-hubs/ about it impacting the price...

Reply:
added this condition based on https://github.com/infracost/infracost/pull/787#discussion_r648733417